web-api/src/business/useCases/caseConsolidation/addConsolidatedCaseInteractor.ts

```diff
@@ -107,7 +107,7 @@ export const determineEntitiesToLock = (
     item => `case|${item}`,
   ),
 });
-
+// 10505: this replicates the existing functionality, but may need to account for `allCasesToConsolidate`?
 export const addConsolidatedCaseInteractor = withLocking(
   addConsolidatedCase,
```

Comment:
This _looks_ like an existing gap in locking; if `caseToUpdate.leadDocketNumber !== caseToConsolidateWith.leadDocketNumber` we may query for more cases that wouldn't be locked. Trying to decide whether to fix that as part of this work